Microsoft.Toolkit.Uwp.UI.Controls/HeaderedTextBlock/HeaderedTextBlock.cs

```diff
@@ -39,6 +41,14 @@ private void UpdateHeader()
             }
         }
 
+        private void UpdateText()
+        {
+            if (_textContent != null)
```

Comment:
I know, but it's still redundant. I see the UpdateHeader method does the same, that's also redundant :)